src/components/DefinitionEntry.js

```diff
@@ -71,16 +71,28 @@ export default class DefinitionEntry extends React.Component {
 
   renderHeadline(definition) {
     const { namespace, name, revision } = definition.coordinates
+    const locationUrl = this.getLocationURL(definition)
```

Comment:
use consistent casing on URL. suggest using Url. I vaguely think that is what we use elsewhere but a quick search in the website and service code will reveal the truth.

Reply:
Url is the correct casing. I looked through website and service code.